jobs/MasterCI/MasterCI

```diff
@@ -66,7 +66,9 @@ node{
             }
         } catch(error){
             // trigger freeze procedure
-            build job: env.MERGE_FREEZER_TRIGGER, parameters: [[$class: 'StringParameterValue', name: 'MANUAL', value:"false"],[$class: 'BooleanParameterValue', name: 'FREEZE_OR_UNFREEZE', value:true]]
+            if (currentBuild.result == "FAILURE"){
+                build job: env.MERGE_FREEZER_TRIGGER, parameters: [[$class: 'StringParameterValue', name: 'MANUAL', value:"false"],[$class: 'BooleanParameterValue', name: 'FREEZE_OR_UNFREEZE', value:true]]
```

Comment:
I think it will conflict with @PengTian0's change now.
@changev  will have to rebase